commit/merkleroot/observation.go

```diff
@@ -60,6 +63,10 @@ func (p *Processor) Observation(
 	}
 
 	if err := p.verifyQuery(ctx, prevOutcome, q); err != nil {
+		if errors.Is(err, ErrSignaturesNotProvidedByLeader) {
+			p.lggr.Infow("RMN signatures not available, returning an empty observation", "err", err)
```

Comment:
So what's the flow at this point? The leader tries to get RMN signatures on the stuff in `prevOutcome`?

Reply:
But this error here is no signatures provided by leader right?